modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -93,6 +98,81 @@ definition {
 		takeScreenshot();
 	}
 
+	@description = "LPS-141820. Verifies violations can be filtered by 'Critical' only."
+	@priority = "5"
+	test CanFilterViolationsByCriticalImpactOnly {
+		property osgi.module.configuration.file.names = "com.liferay.frontend.js.a11y.web.internal.configuration.FFA11yConfiguration.config";
+		property osgi.module.configurations = "enable=B&quot;true&quot;";
+
+		// Workaround for side panel sometimes not displaying LPS-141442
+
+		while (IsElementNotPresent(locator1 = "A11yTool#VIOLATION_PANEL_HEADER")) {
+			Refresh();
+		}
+
+		task ("Verify page has Minor, Moderate, Serious, and Critical set of violations") {
+			for (var severity : list "minor,moderate,serious,critical") {
+				AssertElementPresent(
+					key_text = "${severity}",
+					locator1 = "A11yTool#VIOLATION_PANEL_ITEM_SEVERITY");
+			}
+		}
+
+		task ("Filter impact by Critical") {
+			Click(locator1 = "A11yTool#FILTER_ELEMENT_ICON");
+
+			AssertElementPresent(locator1 = "A11yTool#FILTER_BY_IMPACT_HEADER");
+
+			Click(
+				key_text = "Critical",
+				locator1 = "A11yTool#FILTER_CHECKBOX");
+
+			Click(locator1 = "A11yTool#FILTER_ELEMENT_ICON");
+		}
+
+		task ("Verify only critical violations are present on the side panel") {
```

Comment:
Change Verify to Assert. 

If we're doing actual test assertions, Assert will be a better term to use that stands out for testers and reviewers.